ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/inband-mgmt-spine1-ztp.cfg

```diff
@@ -0,0 +1,128 @@
+!
+no enable password
+no aaa root
+!
+daemon TerminAttr
+   exec /usr/bin/TerminAttr -cvaddr=apiserver.arista.io:443 -cvauth=token-secure,/tmp/cv-onboarding-token -cvvrf=MGMT -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent -taillogs
+   no shutdown
+!
+dhcp server
+   dns server ipv4 1.1.1.1 8.8.8.8
+   tftp server file ipv4 https://www.arista.io/ztp/bootstrap
+   !
+   subnet 172.16.255.0/31
+      !
+      range 172.16.255.1 172.16.255.1
+      name inband ztp for inband-mgmt-parent-Ethernet1
+      default-gateway 172.16.255.0
+   !
+   subnet 172.16.255.2/31
+      !
+      range 172.16.255.3 172.16.255.3
+      name inband ztp for inband-mgmt-parent-vrf-Ethernet1
+      default-gateway 172.16.255.2
+   !
+   vendor-option ipv4 NTP
```

Comment:
please try to use regular option 42 instead of vendor option

Reply:
This will be kept for now, since regular options require EOS 4.31.0, which we would prefer not to require at this point.